tests/Microsoft.Graph.DotnetCore.Core.Test/Requests/ResponseHandlerTests.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.Graph.DotnetCore.Core.Test.Requests
 
     public class ResponseHandlerTests
     {
-        [Fact]
+        [Fact(Skip = "Service Library needs to support System.Text.Json Attributes")]
```

Comment:
> [Fact(Skip = "Service Library needs to support System.Text.Json Attributes")] [](start = 7, length = 78)

is turning this back on tracked by an issue?

Reply:
Yes. This is tracked with this issue https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/issues/80